lib/runtime/imports.go

```diff
@@ -749,6 +749,11 @@ func ext_local_storage_set(context unsafe.Pointer, kind, key, keyLen, value, val
 	log.Warn("[ext_local_storage_set] Not yet implemented.")
 }
 
+// RegisterImports registers the wasm imports for the old version of the substrate test runtime.
+func RegisterImports() (*wasm.Imports, error) {
```

Comment:
Do we need both `RegisterImports` and `registerImports` ? If so, maybe add comments to explain and a comment for `registerImports`, or maybe rename one of them?

Reply:
yeah I'll just rename them

Reply:
hmm, actually I'm thinking of just exposing `NewTestRuntime` but changing it to `NewRuntimeVersion` or something like that, where you can specify the runtime you want as a string, and it will use the correct imports func for you. what do you think?